router/http/src/main/scala/io/buoyant/router/http/MethodAndHostIdentifier.scala

```diff
@@ -1,9 +1,11 @@
 package io.buoyant.router.http
 
+import com.twitter.finagle.http.Request
 import com.twitter.finagle.{Dtab, Path, http}
 import com.twitter.finagle.buoyant.Dst
```

Comment:
might consider just refering to http.Request in the code, since we refer to http.Version (or import Version, etc)